db/schema.rb

```diff
@@ -10,11 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170804092444) do
+ActiveRecord::Schema.define(version: 20170810074052) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "allocations", force: :cascade do |t|
```

Comment:
This is different to the schema for Audits. I think we should be consistent in how we model User relationships.

```rb
create_table "audits", id: :serial, force: :cascade do |t|
  t.string "content_id", null: false
  t.string "uid", null: false
  t.datetime "created_at", null: false
  t.datetime "updated_at", null: false
  t.index ["content_id"], name: "index_audits_on_content_id", unique: true
  t.index ["uid"], name: "index_audits_on_uid"
end
```